frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx

```diff
@@ -134,11 +135,15 @@ export function MissionQueueView() {
         })
     }, [missionQueue, ongoingMissions])
 
-    const missionQueueDisplay = missionQueue
-        .filter((m) => m.installationCode?.toLocaleLowerCase() === installationCode.toLocaleLowerCase())
-        .map((mission, index) => (
-            <MissionQueueCard key={index} order={index + 1} mission={mission} onDeleteMission={onDeleteMission} />
-        ))
+    useEffect(() => {
+        setMissionsToDisplay(
```

Comment:
could this (repeated) code be written into a module?

Reply:
Is it worth making a module for so few lines?

Reply:
Not sure - if it is not a lot of extra effort i would say it is